packages/fxa-auth-server/lib/payments/paypal/client.ts

```diff
@@ -504,7 +504,7 @@ export class PayPalClient {
 
     if (options.refundType === RefundType.partial) {
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      data.AMT = options.amount!.toString();
+      data.AMT = (options.amount! / 100).toFixed(2);
```

Comment:
Should this be [`Number#toFixed()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) or [`Number#toLocaleString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString), or maybe even [`Intl.NumberFormat#format()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/format)?

Wondering if `.toFixed()` is locale-aware enough to know if it should use comma or period separators.

---

```js
const number = 12345678;
const formatter = new Intl.NumberFormat("de-DE", { style: "currency", currency: "EUR" });

console.log((number / 100).toString(), "\t[toString()]");
console.log((number / 100).toFixed(2), "\t[toFixed(2)]");
console.log((number / 100).toLocaleString(), "\t[toLocaleString()]");
console.log(formatter.format(number / 100), "\t[Intl.NumberFormat]");

/*
123456.78 	[toString()]
123456.78 	[toFixed(2)]
123,456.78 	[toLocaleString()]
123.456,78 € 	[Intl.NumberFormat]
*/
```

Reply:
Since this is going to paypal, I don't think any of this is necessary. We aren't working with the currency or locale in the paypal issueRefund method. We are just working with the stripe invoice, the paypal transaction id, and the amount.